### PR TITLE
Chore/update cli plugin plugins

### PR DESCRIPTION
**Pull request by williamads**
#### What is the purpose of this pull request?
To update @vtex/cli-plugin-plugins to 1.13.2.

#### What problem is this solving?
Changes in [#9](https://github.com/vtex/cli-plugin-plugins/pull/9) don't work in toolbelt without this.

#### How should this be manually tested?
The feature fixed in @vtex/cli-plugin-plugins@1.13.2 can be tested with the following command: `vtex-test submit`.

#### Types of changes
- [ ] Refactor (non-breaking change that only makes the code better)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Chores checklist
- [x] Update `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
```
Comment (valentino-amadeus) [inline CHANGELOG.md:10]:
```suggestion
### Updated
```

**Review by valentino-amadeus — APPROVED** [review]
You'll have to wait for https://github.com/vtex/cli-plugin-plugins/pull/9 to be merged.

File: CHANGELOG.md (hunk 1 above)
Comment (gris) [reply CHANGELOG.md:10]: The keepachangelog format doesn’t support this headline. LGTM as is

**Review by gris — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (valentino-amadeus) [reply CHANGELOG.md:10]: Oh, we have some mistakes in CHANGELOG.md, then. 😳

Comment (gris) [reply CHANGELOG.md:10]: What is done is done. Let’s keep in mind next times.

**Pull request merged**